### PR TITLE
Add support for Firebrick Hardware

### PR DESCRIPTION
**Pull request by lewisvive**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ Yes] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/os/firebrick.inc.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+//SNMPv2-MIB::sysDescr.0 = STRING: EdgeSwitch 48-Port 750W, 1.1.2.4767216, Linux 3.6.5-f4a26ed5
+//SNMPv2-MIB::sysDescr.0 = STRING: USW-24P-250, 3.3.15.3976, Linux 3.6.5
+//SNMPv2-MIB::sysDescr.0 = STRING: EdgeSwitch 24-Port 250W, 1.6.0.4900860, Linux 3.6.5-f4a26ed5, 1.0.0.4857129
+//SNMPv2-MIB::sysDescr.0 = STRING: EdgePoint Switch 16-Port, 1.6.0.4900860, Linux 3.6.5-f4a26ed5, 1.0.0.4857129
```
Comment (TheGreatDoc) [inline includes/polling/os/firebrick.inc.php]: Please, clean all unrelated lines

File: includes/definitions/firebrick.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+icon: firebrick
+group: firebrick
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - sysDescr_regex:
+        - '/^FB2900/'
+        - '/^FB2700/'
```
Comment (TheGreatDoc) [inline includes/definitions/firebrick.yaml]: Does they have a valid sysObjectID? Maybe its better than using sysDescr_regex.

Comment (lewisvive) [reply includes/definitions/firebrick.yaml]: Only the generic description ID.

**Comment by lewisvive** [comment]
Changes done

File: html/pages/device/showconfig.inc.php [hunk 3]
```diff
@@ -327,7 +327,7 @@
         // $geshi->set_line_style('color: #999999');
         echo '<div class="config">';
         echo '<input id="linenumbers" class="btn btn-primary" type="submit" value="Hide line numbers"/>';
-        echo htmlspecialchars_decode($geshi->parse_code());
+        echo $geshi->parse_code();
```
[the remark this answers is not in the document]
Comment (lewisvive) [reply html/pages/device/showconfig.inc.php]: I didn't mean to put in this change. We have an issue that XML data from Oxidized is showing as HTML, not as text to screen.

[the remark this answers is not in the document]
Comment (laf) [reply html/pages/device/showconfig.inc.php]: If you can remove it that would be great.

**Comment by laf** [comment]
@lewisvive Any update on this?
